mindmeld/system_entity_recognizer.py

```diff
@@ -83,7 +87,7 @@ def get_response(self, data):
         url = get_system_entity_url_config(app_path=self.app_path)
```

Comment:
``def get_response(self, data, timeout=None): 
        if not timeout:
            timeout = SYS_ENTITY_REQUEST_TIMEOUT
``

Reply:
I am unclear why this needs to be done.